src/ensembl/src/content/app/species/components/species-main-view/SpeciesMainViewStats.tsx

```diff
@@ -0,0 +1,173 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+import ViewInAppPopup from 'src/shared/components/view-in-app-popup/ViewInAppPopup';
+import SpeciesStats from 'src/content/app/species/components/species-stats/SpeciesStats';
+import ExpandableSection from 'src/shared/components/expandable-section/ExpandableSection';
+
+import {
+  getActiveGenomeId,
+  getActiveGenomeStats
+} from 'src/content/app/species/state/general/speciesGeneralSelectors';
+import { getGenomeExampleFocusObjects } from 'src/shared/state/genome/genomeSelectors';
+import { fetchStatsForActiveGenome } from 'src/content/app/species/state/general/speciesGeneralSlice';
+
+import { RootState } from 'src/store';
+import { ExampleFocusObject } from 'src/shared/state/genome/genomeTypes';
+import { GenomeStats } from '../../state/general/speciesGeneralSlice';
+import {
+  StatsSection,
+  sectionGroupsMap
+} from '../../state/general/speciesGeneralHelper';
+
+import styles from './SpeciesMainView.scss';
+
+type Props = {
+  activeGenomeId: string | null;
+  genomeStats: GenomeStats | undefined;
```

Comment:
Why would it ever be undefined? Does it make sense for a component responsible for displaying stats to receive an undefined instead of stats?